galore/__init__.py

```diff
@@ -26,12 +26,13 @@
 from collections.abc import Sequence
 import logging
 
+import galore.formats
 from math import sqrt, log
 import numpy as np
 from scipy.interpolate import interp1d
 
-import galore.formats
-from galore.cross_sections import cross_sections_info
+
+from galore.cross_sections import get_cross_sections, cross_sections_info
```

Comment:
I don't understand why this is included in this PR; it's already part of the master branch. Is your master branch (origin) behind the main one (upstream)? This is also presumably the cause of the merge conlict.